src/models/AssetAttributes.ts

```diff
@@ -0,0 +1,69 @@
+import { Service, ServiceType } from '../ddo/Service'
+import { EncryptionMethod } from '../services/metadata/MetadataService'
+import { MetaData } from "../ddo/MetaData"
+import AssetPrice from './AssetPrice'
+
+
+export class AssetAttributes {    
+
+    static DEFAULT_ENCRYPTION_METHOD = 'PSK-RSA' as EncryptionMethod
+
+    /**
+     * Metadata describing the asset
+     * @see {@link MetaData}
+     */
+    metadata: MetaData    
```

Comment:
I don't think the user should construct the metadata object

Reply:
I think the Metadata class encapsulates this pretty well. A different thing is if we want to provide utility functions in the `Metadata` class to decorate the object creation. To be discuss later